.travis.yml

```diff
@@ -16,19 +14,13 @@ jobs:
       name: Lint JavaScript
     - script: yarn test
       name: Tests
-    # build only for master and devel (tmp: "improvement/semantic-release").
-    - stage: build
-      if: branch IN ("master", "devel")
+    - stage: Build & Publish
```

Comment:
This "stage" is triggered only on the `master` branch.

First, It executes the "script" to build the package (`yarn`, then `yarn build`).
Then "deploy" with the `npx semantic-release` script.